types/contract.go

```diff
@@ -0,0 +1,13 @@
+// Package types
+package types
+
+//Contract define simple information about a SMC in kardia system
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>commentFormatting: put a space between `//` and comment text (gocritic)